charts/incubator/dumbassets/values.yaml

```
@@ -0,0 +1,39 @@
+image:
+  repository: dumbwareio/dumbassets
+  pullPolicy: IfNotPresent
+  tag: 1.0.11@sha256:b12310240508ee26acd4adbbf4b3e42303a4c5918b094daf0d225b3ae00cad93
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+service:
+  main:
+    ports:
+      main:
+        port: 3000
```

```suggestion
service:
  main:
    ports:
      main:
        port: 3000
        targetPort: 3000
```

the software doesnt have the option to change the port with a variable so we have to set a target port